mods/decorations/init.lua

```diff
@@ -0,0 +1,305 @@
+decorations = {}
+
+decorations.register_stalagmites = function(base_node, drop)
+	local base_definition = minetest.registered_nodes[base_node]
+	local sname = string.match(base_node, ':(.*)')
+	local base_description = base_definition.description
+	local groups = {} -- Simply assigning the value of base node's group to here will then make any edits also mirror on the original node (bug?)
+	groups.cracky = base_definition.groups.cracky
+	groups.crumbly = base_definition.groups.crumbly
+	groups.stone = base_definition.groups.stone
+	groups.level = base_definition.groups.level
+	--groups.attached_node = 1
```

Comment:
Its passed wrong somewhere back in the C code, and when you assign here in lua you can change the original by changing the new assigned value. Kinda like something back there is just passed instead of copied

Reply:
Right, but the point is you aren't changing any values here afaik? You are just copying them over.

Reply:
Sure, I don't think that attached_node is going to be implemented because of how those stalagmites work now.